editor/src/clj/editor/disk.clj

```diff
@@ -304,19 +304,24 @@
   (disk-availability/push-busy!)
   (future
     (try
-      (let [hook-opts {:output-directory (get bob-options "bundle-output")
+      (let [invoke-bundle-hooks (boolean (some #(= "bundle" %) bob-commands))
+            hook-opts {:output-directory (or (get bob-options "bundle-output")
+                                             (get bob-options "output")
+                                             "build/default")
                        :platform (get bob-options "platform")
-                       :variant (get bob-options "variant")}]
+                       :variant (get bob-options "variant" "release")}]
```

Comment:
Now we need to provide defaults to hook opts because they are not guaranteed to be present.